src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -248,10 +245,71 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             StartingDirectoryUseParentCheckbox().IsChecked(true);
         }
 
+        // Subscribe to some changes in the view model
+        // These changes should force us to update our own set of "Current<Setting>" members,
+        // and propagate those changes to the UI
+        _ViewModelChangedRevoker = _State.Profile().PropertyChanged(winrt::auto_revoke, [weakThis{ get_weak() }](auto&&, const PropertyChangedEventArgs& args) {
```

Comment:
Not sure if I was pointing at the right PR/commit, but: no need for the weak ref here. Revoker guarantees we'll be alive or that it will be unregistered.

Reply:
then you can drop `page->` off everything inside.

Reply:
just capture `this` by value